CPAN/buildme.sh

```diff
@@ -420,9 +534,9 @@ function build_module {
         
         $PERL_BIN Makefile.PL INSTALL_BASE=$PERL_BASE $makefile_args
         if [ $local_run_tests -eq 1 ]; then
-            $GNUMAKE test
```

Comment:
Do we even need GNUMAKE and MAKE any more? After all this back and forth they're now the same anyway, and GNUMAKE isn't even used any more?

Reply:
@mherger Yes, we need $MAKE calls, for FreeBSD, at least. Tested on FreeBSD 7.2 (bash 4.2.20) and FreeBSD 11.1 (bash 4.4.12). The MAKE and GNUMAKE variable export does nothing to influence what happens when "make" is called the exported variable value is ignored. On the other hand, since $MAKE points to "make" on Linux or "gmake" on FreeBSD, this seems to have the most predictable behavior. I don't know enough about other platforms to know what the utility of GNUMAKE variable declaration is.

For the record, FreeBSD's make still sucks.

Reply:
But what about GNUMAKE? We do define it, but don't use it. Would it be called in some build script outside this one?

Reply:
I don't believe GNUMAKE has any utility. From what I can see with other projects (android-sdk, etc.), defining GNUMAKE is most often done outside the script, and then the script can use it to do the building. So, it could be all GNUMAKE or all MAKE, but I doubt we need both. Which do you prefer?

Reply:
MAKE

Reply:
Okay. Tonight I will: merge public/7.9 in again, add another commit to completely remove GNUMAKE, and do the PR over in platform.